lib/rubocop_director/commands/generate_config.rb

```diff
@@ -7,6 +7,7 @@ class GenerateConfig
       include Dry::Monads::Do.for(:run)
 
       RUBOCOP_TODO = ".rubocop_todo.yml"
+      RUBOCOP_DIRECTOR_CONFIG =  ".rubocop-director.yml"
```

Comment:
We have it here already https://github.com/DmitryTsepelev/rubocop_director/blob/master/lib/rubocop_director.rb#L7

Reply:
Thanks